modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -2351,7 +2361,10 @@ void GDScriptAnalyzer::reduce_identifier_from_base(GDScriptParser::IdentifierNod
 				result.enum_type = name;
 				p_identifier->set_datatype(result);
 			} else {
-				push_error(vformat(R"(Cannot find value "%s" in "%s".)", name, base.to_string()), p_identifier);
+				// Consider as a Dictionary
```

Comment:
Maybe a _TODO_ based on what you discussed at #45558 would help catching this as not fully solved.

Reply:
I dont know enough "git" to ammend a previous commit in the branch 😅 but i will put this if i am asked to squash everything

Reply:
I'm not super fond of this because it looks to me like it would allow arbitrary changes to an enum that supposedly should be constant. While it's a dictionary behind the curtain, it shouldn't be used like so.

Although I do want to do some changes on how enum works, so we could merge it like this for now and I change it later when I get to it.